python/dgl/graphbolt/minibatch.py

```diff
@@ -227,7 +227,7 @@ def blocks(self):
                         sampled_csc.indices,
                         torch.arange(
                             0,
-                            sampled_csc.indptr[-1],
+                            len(sampled_csc.indices),
```

Comment:
what's the difference?

Reply:
You can check #6862 and #6863. The old one initiates a device-to-host transfer (from GPU to CPU), the new one does not initiate any such transfer. This is because `indptr` is on the CUDA device when sampling on the GPU.

Reply:
I updated the links in #6862 since GitHub stopped rendering the images I had attached for some reason.